web_src/css/shared/issuelist.css

```
@@ -54,108 +77,49 @@
   word-break: break-all;
 }
 
-.issue.list > .item .labels-list {
-  position: relative;
-  top: -1.5px;
+.issue.list > .item .title {
+  color: var(--color-text);
+  word-break: break-word;
```

```suggestion
  overflow-wrap: anywhere;
```

`word-break: break-word` is deprecated per [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#break-word).